tests/run_tests.py

```diff
@@ -767,6 +782,8 @@ def replace_global_values(gp, replace):
     gpl = []
     for line in iter(gp.splitlines()):
         line_list = line.split()
+        if line_list == []:
+            continue
```

Comment:
Haha sorry I think actually I originally added these two lines... and yes they are for skipping empty lines (otherwise the function breaks at empty lines). Good suggestion.